README.md

```diff
@@ -1,2 +1,29 @@
-# leadconduit-cakefile
-A Node.JS utility module for building LeadConduit integrations
+# Common LeadConduit Integration Cakefile
+
+A Node.JS utility module for building LeadConduit integrations.
+
+[![Build Status](https://travis-ci.org/activeprospect/leadconduit-cakefile.svg?branch=master)](https://travis-ci.org/activeprospect/leadconduit-cakefile)
+
+
+This module provides the standard build utilities for integration modules. To use it:
+
+1. require in your module's `package.json` (of course)
+2. existing modules may have `coffee-script` and `mocha` in their `DevDependencies` - those can (probably) be removed.
+3. create a file called `Cakefile` in the root of the module, with this single line:
+
+```coffeescript
+require('leadconduit-integration').cake(task)
+```
```

Comment:
I'm honestly a little hand-wavy on how this works, but I can attest that it does work as I wrote it. I'm not sure how (or why) to eliminate the `cake` function, nor do I see how your first question omits the `(task)` part but your block example doesn't.